src/Gui/ToolBarManager.cpp

```diff
@@ -202,6 +204,12 @@ void ToolBarManager::setup(ToolBarItem* toolBarItems)
                 QApplication::translate("Workbench",
                                         toolbarName.c_str())); // i18n
             toolbar->setObjectName(name);
+            if (nameAsToolTip){
+                std::ostringstream tooltip;
+
+                tooltip << "[" << toolbarName << "]";
+                toolbar->setToolTip(QApplication::translate("Workbench", tooltip.str().c_str()));

```

Comment:
This doesn't work because you introduce a new string to be translated first. Change your UI language to see that.
The aim is to reduce the translation work, therefore we should take the name that is already translated and concatenate the brackets around it.